code/modules/mob/living/carbon/human/species.dm

```diff
@@ -465,6 +465,9 @@
 	H.voice_filter = initial(H.voice_filter)
 	H.health_threshold_crit = -50
 
+#define CLONELOSS_THRESHOLD 0
```

Comment:
This should probably be the same threshold as a human being in crit (-50).
While I like the concept, punishing robots for being at 0 hp, when a human is perfectly capable of existing at 0 hp is a bit too much.
The idea being that if you are at/under critical threshold for too long, your chassis will degrade to the point that you will die from it.